open-craft/ansible-common-server
# Install the New Relic monitoring agent

This adds `nrsysmond` (New Relic Servers) to all our hosts. They will report with their hostname. No labels are defined.
There's an option to toggle it off, e.g. for development machines, though another option is to let them report and then set up no alerts for them.
The variable names start with `COMMON_SERVER_` to avoid conflicts.
There are some variables that could be deleted because we don't need to ever configure them.

To test this:
- this role has been used to provision rabbitmq-dev, and now it reports here: https://rpm.newrelic.com/accounts/1346955/servers/107502564
- test it in another server
- you can check out `ansible-secrets` (but add the NR key to it, i.e. https://github.com/open-craft/ansible-secrets/pull/9), update the roles, then replace the `common-server` one with a directory with this branch, then continue deploying
- in the end, `nrsysmond` should be running and it should report to the New Relic web interface
